### ISSUE TITLE
How should I set the parameters of CriteoDataset()?

### ISSUE THREAD

**Issue by guvcolie** [facebookresearch/dlrm#78]
I download the raw kaggle criteo dataset, there's a readme.txt, train.txt and test.txt. 
1. How can  I get the training set, val set and test set by your dlrm_data_pytorch.py? 
2. And how do I set the raw_path, pro_data and memory_map hyper-params when training?
Thank you!

**Comment by mnaumovfb** [comment]
Please take a look at the README "Benchmarking" section, where in point 2. we discuss the interface to Criteo Kaggle DAC dataset. You will notice that we use a shell script `./bench/dlrm_s_criteo_kaggle.sh`, which specifies these parameters for you. You can also pass additional parameters in quotes afterwards, such as `./bench/dlrm_s_criteo_kaggle.sh "--test-freq=1024 --memory-map"`.

**Comment by guvcolie** [comment]
Thank you!  If I want "In experiments, typically the 7th day is split into a validation and test set while the first 6 days are used as the training set. ", I must set the hyper-param randomize="day", right?

**Comment by mnaumovfb** [comment]
I recommend leaving the default --data-randomize="total", you will still get the above split.

**Comment by guvcolie** [comment]
OK, got it! thank you!
